cmake/MapnikExportPkgConfig.cmake

```diff
@@ -3,7 +3,7 @@ function(create_pkg_config_file _target _lib_name _description)
 prefix=@CMAKE_INSTALL_PREFIX@
 exec_prefix=${prefix}
 includedir=${prefix}/include
-libdir=${exec_prefix}/lib
+libdir=${exec_prefix}/@CMAKE_INSTALL_LIBDIR@
```

Comment:
as @hummeltech  has written, I would prefer `MAPNIK_LIB_DIR` to be used

Reply:
👍